app-settings.json

```diff
@@ -0,0 +1,28 @@
+[
```

Comment:
This doesn't need to be merged - it is auto-created by the settings plugin and not used. I'm going to merge + delete.